src/models/document/document-types.ts

```diff
@@ -52,6 +52,8 @@ export const DocumentTypeEnum = types.enumeration("type",
 export type DocumentType = Instance<typeof DocumentTypeEnum>;
 export type ProblemOrPlanningDocumentType = typeof ProblemDocument | typeof PlanningDocument;
 export type OtherDocumentType = typeof PersonalDocument | typeof LearningLogDocument;
+export type ProcessableDocumentTypes =
```

Comment:
What is the type used for? I can't think of a place in the code where we'd only want to use this for static typing the allowed documents.

Reply:
Thanks, I think at an early point I must have thought I needed this at some point to expand the list of things that the db store functions could handle? Anyway, removing it.